### PR TITLE
Some cleanup

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

This pr cleans up some left over TODOs

## Proposed changes

Added inventory blocks tag
Added tile entities tag
Added storage blocks tag
change air check to .isAir()

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.14.* - 1.19.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Review by svr333 — APPROVED** [review]
lgtm

**Review by svr333 — DISMISSED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoUtils.java
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoUtils.java:72]: This tag does not necessarily represent all storage blocks. Only the ones cargo supports. The tag name should reflect that.

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/autocrafters/AbstractAutoCrafter.java
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/autocrafters/AbstractAutoCrafter.java:205]: This tag does not necessarily represent all storage blocks. Only the ones Auto Crafters support. The tag name should reflect that.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoUtils.java
Comment (J3fftw1) [reply src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoUtils.java:72]: I thought let me generalize them so it can be used on multiple places.
At first it was like you wanted XD

File: src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java:275]: These comments need to be updated.
Also some consistency in naming would probably be better, e.g. "XYZ_SUPPORTED_STORAGE_BLOCKS"

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/Crucible.java
Comment (J3fftw1) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/Crucible.java:202]: You wont me to change this or just let it be blocked till we drop support?
There is no priority in this anyways.

Comment (svr333) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/Crucible.java:202]: the pr to drop 1.14 is already open so we can just delay this one until that one is merged

File: src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java:274]: Please update these docs

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java:670]: This is marked as non-null.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java
Comment (J3fftw1) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java:670]: This wasnt supposed to be in here whoops

**Review by svr333 — APPROVED** [review]
did you change line endings on the last commit? entire file has whitespace edits

anyway the code looks good, cant merge until we drop 1.14 support though

**Comment by J3fftw1** [comment]
This isnt blocked anymore because 1.14 and 1.15 arent supported anymore

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**
